Gopkg.toml

```diff
@@ -158,3 +158,7 @@ required = [
 [[constraint]]
   name = "github.com/openshift/api"
   revision = "0d921e363e951d89f583292c60d013c318df64dc"
+
+[[constraint]]
```

Comment:
With the subvendor repo I think there's now some rule about dependencies have to be updated independent of code changes but I'm not sure if this means the toml file is separate or only the actual upload of the vendor content.

Reply:
Can you include junit report in the base image used by prow - like it is done in circleci ?

Reply:
yes matt is correct, you need to rebase, rm your vendor dir if it's not the git submodule backed one
do make pull (or make submodule-sync)
then run `dep ensure` and cd in vendor/ and make a PR for the changes first
https://github.com/istio/istio/wiki/Vendor-FAQ#how-do-i-add--change-a-dependency
I'm glad someone (beside me) will help test/confirm that process

Reply:
Junit results harness has moved to makefile so circle ci also benefits from this.

Tool chain dependencies are captured with a version-tracked base docker image. Here is my other PR updating this image per request by Costin https://github.com/istio/test-infra/pull/698/files